protocol-units/execution/opt-executor/Cargo.toml

```diff
@@ -69,7 +68,7 @@ aptos-indexer-grpc-table-info = { workspace = true }
 aptos-indexer = { workspace = true }
 aptos-protos = { workspace = true }
 aptos-logger = { workspace = true }
-
+tonic = { workspace = true }
```

Comment:
This dependency does not seem to be needed.

Reply:
This was from trying to see if the result of `git bisect` was indeed valid.

Reply:
It was not.